src/gui/optionsdlg.cpp

```diff
@@ -174,6 +174,9 @@ OptionsDialog::OptionsDialog(QWidget *parent)
     m_ui->groupFileAssociation->setVisible(false);
 #endif
 
+    m_ui->textWebUIRootFolder->setMode(FileSystemPathEdit::Mode::DirectoryOpen);
+    m_ui->textWebUIRootFolder->setDialogCaption(tr("Choose Alternative UI files location"));
```

Comment:
`Choose alternative Web UI folder`, as the files are always placed in some specific folder right?

Reply:
I use here the wording proposed by some another reviewer (@evsh, unless I'm wrong). I would prefer to leave it as is (at least in this PR).